lerna.json

```diff
@@ -1,6 +1,6 @@
 {
   "packages": [
-    "config/*",
+    "configs/*",
```

Comment:
I don't know why so many things were working before, but this was definitely a typo (see also [Slack thread](https://mongodb.slack.com/archives/GJ2CBBY86/p1705983400964289))